instrumentation/jms/README.md

```diff
@@ -98,6 +98,10 @@ void process(Message message) {
 }
 ```
 
+## Compatibility issues
+
+* There are known issues with ActiveMQ Client versions <= `5.15.11` when using `BytesMessage`: <https://github.com/openzipkin/brave/issues/967>. ActiveMQ has [fixed this issue](https://issues.apache.org/jira/browse/AMQ-7291) and following versions should work when using `BytesMessage`.
```

Comment:
```suggestion
* There are known issues with ActiveMQ Client versions <= `5.15.13` when using `BytesMessage`: <https://github.com/openzipkin/brave/issues/967>. ActiveMQ has [fixed this issue](https://issues.apache.org/jira/browse/AMQ-7291) and following versions should work when using `BytesMessage`.
```

per the issue I think this won't be fixed until 5.15.13 now..

Reply:
:+1:  crossing fingers

Reply:
countdown till 5.16!

Reply:
just mentioning 5.16 is not out yet. we can check in another couple weeks I guess

Reply:
https://github.com/apache/activemq/releases/tag/activemq-5.16.0 :) waiting for artifacts to be available on maven central.